### PR TITLE
Enrich English declarative Doc

### PR DESCRIPTION
**Pull request by zhhsplendid**
### PR types

Others
### PR changes

Docs
### Describe

Enrich Declarative Doc

### CODE REVIEW

**Review by Aurelius84 — DISMISSED** [review]

File: python/paddle/fluid/dygraph/jit.py [hunk 1]
```diff
@@ -132,7 +132,10 @@ def _declarative_(dygraph_func):
     """
     Converts imperative dygraph APIs into declarative function APIs. Decorator
     @declarative handles the Program and Executor of static mode and returns
-    the result as a dygraph VarBase.
+    the result as a dygraph VarBase. Users could use the returned dygraph
+    Varbase to do imperative training, inference, or other operations. If the
```
Comment (liym27) [inline python/paddle/fluid/dygraph/jit.py]: 1. The concept of `VarBase` is not exposed to users. Better to use `Tensor ` instead.
2. The returned result may contain more than one Tensor.

Comment (zhhsplendid) [reply python/paddle/fluid/dygraph/jit.py]: Done

**Review by liym27 — DISMISSED** [review]
LGTM

**Review by Aurelius84 — APPROVED** [review]

**Review by liym27 — APPROVED** [review]

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**